.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
```

Comment:
This looks to be added accidentally.

Reply:
My apologies.  I'm still new to GitHub, and in trying to add support for my static analyzer of choice to my local fork, I pushed it as well.  It can be disregarded.

Reply:
You need to remove it, or your PR cannot be merged.

Reply:
It has been removed.